src/index.js

```diff
@@ -49,11 +80,23 @@ const css = (...styleDefinitions) => {
         return "";
     }
 
-    const className = validDefinitions.map(s => s._name).join("-o_O-");
+    // Filter out "plain class name" arguments, which just want us to add a
+    // classname to the end result, instead of generating styles.
```

Comment:
I think it's probably helpful to say `this is the case, e.g., for descendent selectors where the relevant CSS is generated by the call to `css(styles.foo)`, not the call to `css(styles.foo.someChild)`.